lib/matplotlib/axes/_axes.py

```diff
@@ -798,15 +798,15 @@ def axhline(self, y=0, xmin=0, xmax=1, **kwargs):
 
         Returns
         -------
-        line : :class:`~matplotlib.lines.Line2D`
+        line : a `.Line2D` instance
```

Comment:
```suggestion
        line : `~matplotlib.lines.Line2D`
```

I know that some other places still have the "instance" thing. But that should be removed there as well. We also wouldn't write `s : a str instance`.

See also https://matplotlib.org/devel/documenting_mpl.html#referencing-types.